File: Problem-6/problem6.php
<?php

/********************
 * 
 * Find the difference between the sum 
 * of the squares of the first one 
 * hundred natural numbers and the 
 * square of the sum.
 * 
 *******************/

$sumSquares = 0;
$sum = 0;

for ($i = 1; $i <= 100; $i++)
{
	$sum += $i;
	$sumSquares += $i * $i;
}

echo ($sum * $sum) - $sumSquares;

File: Problem-6/problem6.py
def sumSquareSquareSumDiff(max):
    sum = 0
    sumOfSquares = 0
    #Python range is exclusive, so we need to add 1 to max
    for i in range(max + 1):
        sum += i
        sumOfSquares += (i ** 2)
    print "The sum of squares for all numbers between 1 and %d is %d, while the square of the sum is %d" % (max, sumOfSquares, sum ** 2)
    print "Therefore, the difference between the sum of squares and the square of the sum is %d" % ((sum ** 2) - sumOfSquares)
    
sumSquareSquareSumDiff(100)    